tests/release/pipelines/README.md

```diff
@@ -1,41 +1,44 @@
 
-### All tests must have the label `release-pipelines` to avoid running them against the dev environment by OpenShift CI
-## prerequisites: 
-   - Export the following environment variables:
-		```
-    	- TOOLCHAIN_API_URL_ENV: Offline token used for getting Keycloak token in order to authenticate against stage/prod cluster
-		- KEYLOAK_URL_ENV:       Keycloak URL used for authentication against stage/prod cluster
-		- OFFLINE_TOKEN_ENV :    Toolchain API URL used for authentication against stage/prod cluster
-		```
-   -  The tests will run on two dedicated namespaces, so a user not part of them need to request access to the following namespaces:
-		```
-		- dev-release-team-tenant
-		- managed-release-team-tenant
-		```
-
-## How to Run tests:
+## Branch alignment
+ * Below is the branches in [release-service-catalog](https://github.com/redhat-appstudio/release-service-catalog/tree/main). All these branches will be tested per requirement.  
+   - production	: this branch will be used for RHTAP product environment
+   - stage	: this branch will be used for RHTAP stage environment
+   - development: this branch is the default branch for development
 
-### 1. To run all e2e-tests in suite ensure your changes of the suite tests are saved.
-- navigate to `e2e-tests` directory 
- ```bash
- make build 
- ./bin/e2e-appstudio --ginkgo.junit-report=report.xml  --ginkgo.focus="pipelines"
- ```
+## Test cases 
+### The good path with pushing to Pyxis stage (push_to_external_registry.go)
 
-### 2. To run specific test we should define the test with specific label.
-[Ginkgo](https://onsi.github.io/ginkgo/#why-ginkgo) uses labels filtering, adding a label for a test will enable to run specific tests with this label. 
+This test is designed to run the release pipeline `push-to-external-registry` and validates that the artifacts are successfully pushed to Pyxis.
 
-For example: 
-If a test has the label `label-test` to a test in suite `pipelines` then the test should run using the following arguments:
-
-- navigate to `e2e-tests` directory 
- ```bash
- make build 
- ./bin/e2e-appstudio --ginkgo.junit-report=report.xml  --ginkgo.focus="pipelines" --ginkgo.label-filter="label-test"
- ```
+Checkpoints:
+  - Two build PipelineRuns are created in the dev namespace.
+  - The build PipelineRuns pass.
+  - The release PipelineRuns are successfully created in the managed namespace.
+  - The release PipelineRuns are expected to pass.
+  - The Releases pass.
+  - Validate that the SBOM for both components were uploaded to Pyxis stage.
 
+### The good path of FBC related tests (fbc_release.go)
 
+This test is designed to run the release pipeline `fbc_release` related test and validates that the artifacts are successfully pushed. There are two test cases currently in this file. One of them is the happy path of releasing FBC related fragments and the other is for testing FBC hotfix releasing.
 
+Prerequisites: 
+   - Export the following environment variables:
```

Comment:
just for my own understanding, these variables are set in the CI runs so they work properly right?

Reply:
Yes